### PR TITLE
Added unit test

### PR DESCRIPTION
**Pull request by Agnes-Kamondi**
# Description

This PR adds tests to ensure the functionality and stability of all the Kolezza sawaTok models. It includes tests for both existing features and any new functionality introduced. 

Fixes # (issue) [Provide a link to the related Jira Task]
https://kamondiagnesauma-1723202712527.atlassian.net/jira/software/projects/SCRUM/boards/1/backlog?epics=visible
## Type of change

Specify the kind of PR/MR

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update
- [ ] Optimization
- [x] Improvement
- [ ] Bug fix

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] Test A - e.g Manually tested user creation
- [ ] Test B

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: kolezzaSawaTok/child_progress/tests.py [hunk 1]
```diff
@@ -1,3 +1,136 @@
+# from django.core.exceptions import ValidationError
+# from django.test import TestCase
```
Comment (Ndichu-shee) [inline kolezzaSawaTok/child_progress/tests.py:2]: why did you comment this tests

File: kolezzaSawaTok/kolezzaSawaTok/settings.py [hunk 2]
```diff
@@ -29,112 +31,137 @@
...
 
 # Internationalization
-LANGUAGE_CODE = 'en-us'
-TIME_ZONE = 'UTC'
+LANGUAGE_CODE = "en-us"
+TIME_ZONE = "UTC"
 USE_I18N = True
 USE_TZ = True
 
+# REST_FRAMEWORK = {
```
Comment (Ndichu-shee) [inline kolezzaSawaTok/kolezzaSawaTok/settings.py:130]: remove this

File: kolezzaSawaTok/kolezzaSawaTok/settings.py [hunk 3]
```diff
@@ -29,112 +31,137 @@
...
+
+SIMPLE_JWT = {
+    'AUTH_COOKIE': 'access_token',  # Cookie name for storing the access token
+    'AUTH_COOKIE_SECURE': False,    # Set to True in production
+    'AUTH_COOKIE_HTTP_ONLY': True,  # HTTP-only cookie to prevent JavaScript access
+    'AUTH_COOKIE_PATH': '/',        # Cookie available site-wide
+    'AUTH_COOKIE_SAMESITE': 'Lax',  # Adjust SameSite settings as needed
+}
+
+# SIMPLE_JWT= {
```
Comment (Ndichu-shee) [inline kolezzaSawaTok/kolezzaSawaTok/settings.py:164]: why did you comment this

File: kolezzaSawaTok/speech_therapist/tests.py [hunk 4]
```diff
@@ -1,3 +1,97 @@
+# # Create your tests here.
+# from django.core.exceptions import ValidationError
```
Comment (Ndichu-shee) [inline kolezzaSawaTok/speech_therapist/tests.py:2]: why did you comment this

Comment (Agnes-Kamondi) [reply kolezzaSawaTok/speech_therapist/tests.py]: Removed all the comments

**Review by priscilla-mikisa — APPROVED** [review]

**Review by queen-bellah — APPROVED** [review]

**Review by Ndichu-shee — APPROVED** [review]

**Review by Mary-Njiru — APPROVED** [review]

**Review by ChristineAkinyi — APPROVED** [review]

**Review by Wachira-glory — APPROVED** [review]

**Pull request merged**